meteor/client/ui/SegmentStoryboard/SegmentStoryboard.scss

```diff
@@ -166,6 +168,21 @@ $break-width: 35rem;
 		background: $segment-timeline-background-color;
 	}
 
+	&--out-of-the-loop {
```

Comment:
Maybe this could be called "--outside-loop"? "out-of-the-loop" could be read as "the out (end) of the loop".